docs/user/dashboard/edit-dashboards.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[edit-dashboards]]
+== Edit dashboards
+
+Now that you have added panels to your dashboard, you can add filter panels to interact with the data, and Markdown panels to add context to the dashboard. 
+To make your dashboard look the way you want, use the editing options.
+
+[float]
+[[add-controls]]
+=== Add controls
+
+To filter the data on your dashboard in real-time, add a *Controls* panel.
+
+You can add two types of *Controls*:
+
+* Options list — Filters content based on one or more specified options. The dropdown menu is dynamically populated with the results of a terms aggregation. 
```

Comment:
How about putting "Options list" and "Range slider" in bold so they stand out a bit?

Reply:
These are not official Kibana names, so I didn't bold them